app/account/components/AccountDetails/components/AvatarUpdate/cropUtils.ts

```diff
@@ -0,0 +1,32 @@
+export const getCroppedImg = async (imageSrc: string, crop: { x: number; y: number; width: number; height: number }, userName: string) => {
+  const avatarMaxSizeInPx = 192
```

Comment:
I don't see this variable being used anywhere in this file. Is it's implementation missing or is it just no longer necessary?

Reply:
Fixed